nomos-services/time/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+edition = "2021"
+license = { workspace = true }
+name    = "nomos-time"
+version = "0.1.0"
+
+[dependencies]
+async-trait        = "0.1"
+cryptarchia-engine = { workspace = true, features = ["serde", "tokio"] }
+futures            = "0.3"
+log                = "0.4.22"
+nomos-tracing      = { workspace = true }
+overwatch-rs       = { workspace = true }
+serde              = { version = "1.0", optional = true, features = ["derive"] }
+services-utils     = { workspace = true }
+sntpc              = { version = "0.5", optional = true }
+thiserror          = { version = "2.0", optional = true }
+time               = "0.3"
+tokio              = { version = "1", features = ["macros", "sync"] }
+tokio-stream       = "0.1"
+url                = "2.5"
+
+[features]
+default = []
+ntp     = ["sntpc/tokio-socket", "thiserror/default"]
```

Comment:
I guess we want to use the `dep:` prefix for these deps, as otherwise we would be exposing two additional features called `sntpc` and `thiserror` which I believe we don't want? https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

Reply:
True. I forgot about this actually. That also removed the need for the default feature. Thanks!